src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -275,7 +275,8 @@ struct MultiPlayerPlayerGroup : public UI::Box {
 		}
 		type_dropdown_.clear();
 		// AIs
-		if (settings.allows_ais(id_)) {
+		if (settings.allows_ais(id_) &&
+		    settings.get_tribeinfo(settings.players[id_].tribe).suited_for_ai) {
```

Comment:
This causes a weird behavior: 
1. Choose Player 1
2. Set tribe to New Tribe
3. Close the slot
4. The tribe is no longer visible and it is not clear why an AI cannot be used

Better force the tribe to a valid one, when an AI is selected.

Reply:
all issues should be fixed now (hopefully)

Reply:
Not really. Now I can choose an AI in the closed slot, but the tribe dropdown does not update
![Bildschirmfoto vom 2021-02-26 21-12-09](https://user-images.githubusercontent.com/23122583/109350159-52c4c280-7877-11eb-85b3-e8abba9f2f22.png)